source/loader/layers/sanitizer/linux/backtrace.cpp

```diff
@@ -17,20 +17,20 @@ namespace ur_sanitizer_layer {
 StackTrace GetCurrentBacktrace() {
     void *Frames[MAX_BACKTRACE_FRAMES];
```

Comment:
Using `BacktraceFrame` type for `Frames`?

Reply:
Done